allennlp/testing/test_case.py

```diff
@@ -93,171 +92,6 @@ def ensure_model_trains_and_loads(self, model_class, args: Params):
                 assert_allclose(model_prediction, loaded_prediction)
         return model, loaded_model
 
-    @staticmethod
-    def one_hot(index, length):
-        vector = numpy.zeros(length)
-        vector[index] = 1
-        return vector
-
-    def write_snli_files(self):
-        with codecs.open(self.TRAIN_FILE, 'w', 'utf-8') as train_file:
-            train_file.write('1\ttext 1\thypothesis1\tentails\n')
-            train_file.write('2\ttext 2\thypothesis2\tcontradicts\n')
-            train_file.write('3\ttext3\thypothesis3\tentails\n')
-            train_file.write('4\ttext 4\thypothesis4\tneutral\n')
-            train_file.write('5\ttext5\thypothesis 5\tentails\n')
-            train_file.write('6\ttext6\thypothesis6\tcontradicts\n')
-        with codecs.open(self.VALIDATION_FILE, 'w', 'utf-8') as validation_file:
-            validation_file.write('1\ttext 1 with extra words\thypothesis1\tentails\n')
-            validation_file.write('2\ttext 2\tlonger hypothesis 2\tcontradicts\n')
-            validation_file.write('3\ttext3\thypothesis withreallylongfakeword\tentails\n')
-
-    def write_sequence_tagging_files(self):
```

Comment:
I need these ones in my other PR for the simple tagger, but I can copy them over if you'd prefer to have these in the respective tests. I'd possibly rather they were in `AllenNlpTestCase` because when we test the models, we're also going to need access to these files, but if you don't mind the duplication I guess no problem.